chaos.py

```diff
@@ -41,7 +41,9 @@ def restart_self():
     """ restart our process """
     os.execl(sys.executable, sys.executable, *sys.argv)
 
-
+def install_requirements():
+    """install or update requirements"""
+    os.system("pip install -r requirements.txt")
```

Comment:
I believe this will execute the system's pip, not the virtualenv's pip.  Strip out the virtualenv's bin directory from `sys.executable` to get that pip.